content/300-guides/300-upgrade-guides/200-upgrading-versions/030-upgrading-to-prisma-4.mdx

```diff
@@ -8,12 +8,14 @@ toc: true
 
 <TopBlock>
 
-Prisma 4 introduces a number of **breaking changes** if you are upgrading from an earlier version (`3.x.x`). This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
+Prisma 4 introduces a number of **breaking changes** if you are upgrading from an Prisma 3 version (`3.x.x`). This guide explains how this upgrade might affect your application and gives instructions on how to handle any changes.
```

Comment:
The same breaking changes exist if you update from Prisma 2.x

Reply:
Done